### DISCUSSION TITLE
UseEqualsAssignmentAsEqualsOperator

### DISCUSSION THREAD

**Discussion by ArdenCarl** [davideicardi/DynamicExpresso#273] [Ideas]
The UseEqualsAssignmentAsEqualsOperator option should let you get or set if the = should instead act like an equal operator instead of ==.

There's another library out there which offers a similar feature:
https://eval-expression.net/options#useequalsassignmentasequalsoperator

**Comment by davideicardi** [comment]
I think it should be possible to implement that option, but in general the idea is to follow C# rules ... There are already some exceptions, for example about case insensitive. Anyway if you want to try to implement that inside a PR we can eval it.
